python-spec/src/somacore/data.py

```diff
@@ -454,15 +467,6 @@ class SparseRead:
 
     __slots__ = ()
 
-    def coos(self) -> ReadIter[pa.SparseCOOTensor]:
```

Comment:
just a note to keep a pin in this depending upon the results of https://github.com/single-cell-data/TileDB-SOMA/pull/952#discussion_r1108896081

(if it does get spuriously removed here, it's not a huge problem; implementations are allowed to provide *extra* methods and features, and we can just re-add it)

Reply:
yeah @thetorpedodog  totally not gonna land this PR until that one is resolved

Reply:
resolved on latest commit: see also
https://github.com/single-cell-data/TileDB-SOMA/pull/952#discussion_r1109051067